precompiles/pallet-xcm/XcmInterface.sol

```diff
@@ -93,4 +101,28 @@ interface XCM {
         uint32 feeAssetItem,
         Weight memory weight
     ) external;
+
+    function transferAssetsUsingTypeAndThenLocation(
+        Location memory dest,
+        AssetLocationInfo[] memory assets,
+        TransferType assetsTransferType,
+        Location memory maybeAssetsRemoteReserve,
+        uint8 remoteFeesIdIndex,
+        TransferType feesTransferType,
+        Location memory maybeFeesRemoteReserve,
+        bytes memory customXcmOnDest,
+        Weight memory weight
+    ) external;
+
+    function transferAssetsUsingTypeAndThenAddress(
+        Location memory dest,
+        AssetAddressInfo[] memory assets,
+        TransferType assetsTransferType,
+        Location memory maybeAssetsRemoteReserve,
+        uint8 remoteFeesIdIndex,
+        TransferType feesTransferType,
+        Location memory maybeFeesRemoteReserve,
+        bytes memory customXcmOnDest,
+        Weight memory weight
```

Comment:
@Agusrodri I think it's better to delete this parameter and always sue Unlimited, as XCM v5 will remove itn the RFC has been mergerd: https://github.com/polkadot-fellows/RFCs/blob/main/text/0105-xcm-improved-fee-mechanism.md

Reply:
Done in cada24d43f985b0996e3a814ddf75a0d796a45eb.